internal/proxy/middleware/authz/authz.go

```diff
@@ -265,10 +267,16 @@ func createResources(permissionAttributes map[string]interface{}) ([]resource.Re
 	}
 
 	for _, res := range resourceList {
+		nsID := namespace.CreateID(backendNamespace[0], resourceType[0])
```

Comment:
Are we sure the backendNamespace and resourceType always have at least 1 element? This will cause panic if not

Reply:
yes, 
- for backend namespace, we have check here: https://github.com/odpf/shield/pull/143/files#diff-389da525f8e810a300976024407d46ceccaa806befa60ab2cfe8da17892916aeR255
- if proxy doesn't catch resource_type, it will be `""`, defined here: https://github.com/odpf/shield/pull/143/files#diff-389da525f8e810a300976024407d46ceccaa806befa60ab2cfe8da17892916aeR106